lapis/src/test/kotlin/org/genspectrum/lapis/model/AdvancedQueryFacadeTest.kt

```diff
@@ -237,26 +221,6 @@ class AdvancedQueryFacadeTest {
         assertThat(resultWords, equalTo(result))
     }
 
-    @Test
-    fun `given a advancedQuery with a 'Maybe' expression THEN returns the corresponding SiloQuery`() {
-        val advancedQuery = "MAYBE(300G)"
-
-        val result = underTest.map(advancedQuery)
-
-        val expectedResult = Maybe(NucleotideSymbolEquals(null, 300, "G"))
-        assertThat(result, equalTo(expectedResult))
-    }
-
-    @Test
-    fun `GIVEN a advancedQuery with a mixed-case 'Maybe' expression THEN returns 'Maybe' SiloQuery`() {
-        val advancedQuery = "maYbE(T12C)"
-
-        val result = underTest.map(advancedQuery)
-
-        val expectedResult = Maybe(NucleotideSymbolEquals(null, 12, "C"))
-        assertThat(result, equalTo(expectedResult))
-    }
-
     @Test
```

Comment:
I find it odd to move the Maybe and not into the parametrized tests but not do the same for the `or` and `and`